node/README.md

```diff
@@ -14,12 +14,16 @@ The release of Valkey GLIDE was tested on the following platforms:
 
 Linux:
 
--   Ubuntu 22.04.1 (x86_64)
+-   Ubuntu 22.04.1 (x86_64 and aarc64)
```

Comment:
```suggestion
-   Ubuntu 22.04.1 (x86_64 and aarch64)
```

Reply:
or arm64?

Reply:
IDK, i like arm more, but what is more known?